config/application.rb

```diff
@@ -33,6 +34,6 @@ class Application < Rails::Application
       g.helper_specs false
       g.fixture_replacement :factory_girl, :dir => 'spec/factories'
     end
-
+    config.autoload_paths << Rails.root.join('lib')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.